NTO/Documents/attributes/documentPart.ttl

```diff
@@ -0,0 +1,14 @@
+@prefix ogit.Documents:        <http://www.purl.org/ogit/Documents/> .
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Documents:documentPart
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "documentPart";
+	dcterms:description "Contains the part of a document.";
```

Comment:
"Contains the part of a document."

What does this mean and what information stored here? If it is part of documents and then its already document (and probably entity instead attribute?)

Reply:
This attribute is required for document info records from SAP. "Part of document" is relevant for larger original documents, which are split into multiple parts. See: https://blog.sap-press.com/how-to-create-a-document-info-record-in-sap-s4hana